styles/style.scss

```diff
@@ -135,7 +138,35 @@ body{
         bottom: 2%;
         left: 7%;
         width: 20%;
+    
+    }
+    &--Back{
```

Comment:
ten button powinien być raczej u dołu strony - w tej chwili jest wyżej - on purpose?

Reply:
jak wrzucam troche niżej to mam efekt scrolla, pokombinuje jeszcze